26.8.2015/css/main.css

```diff
@@ -38,3 +38,34 @@ header .signUp:focus{
   background-color: transparent;
 }
 /* Banner Will Start Here */
+.banner{
+  padding: 300px 0;
+  background-color: #666666;
+}
+.play{
+  background-image: url(../img/icons.png);
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  width: 130px;
+  height: 130px;
+}
+.play,
+h1{
+  color: #fff;
+}
+h1{
+  position: relative;
+  font-size: 60px;
+  font-weight: normal;
+}
+.banner a{
+  position: absolute;
+  top: 496px;
```

Comment:
why giving this much top property? use parent to give relative, using this much value for the top is good practice